src/applications/vaos/appointment-list/components/UpcomingAppointmentsList.jsx

```diff
@@ -4,85 +4,45 @@ import { shallowEqual, useDispatch, useSelector } from 'react-redux';
 import recordEvent from 'platform/monitoring/record-event';
```

Comment:
Fix eslint warning.
import { recordEvent } from '@department-of-veterans-affairs/platform-monitoring/exports';